xds/internal/xdsclient/tests/fallback_test.go

```diff
@@ -154,10 +154,16 @@ func (s) TestFallback_OnStartup(t *testing.T) {
 		t.Fatalf("Failed to create bootstrap file: %v", err)
 	}
 
-	// Create an xDS client with the above bootstrap configuration.
-	xdsC, close, err := xdsclient.NewForTesting(xdsclient.OptionsForTesting{
-		Name:     t.Name(),
-		Contents: bootstrapContents,
+	// Create an xDS client with the above bootstrap configuration and a short
+	// idle channel expiry timeout. This ensures that connections to lower
+	// priority servers get closed quickly, for the test to verify.
```

Comment:
We don't have the idle channel expiry timeout anymore. So this comment seems to be out of date.

Reply:
Same applies to other comments in this test.